.pipelines/windows-buddy-main.yml

```diff
@@ -0,0 +1,127 @@
+
+pr:
+- main
+
+trigger:
+- none
+
+parameters: # parameters are shown up in ADO UI in a build queue time
+- name: 'debug'
+  displayName: 'Enable debug output'
+  type: boolean
+  default: false
+
+variables:
+  Codeql.Enabled: true
```

Comment:
Do we need to enable codeql twice?

Reply:
Left that in by mistake, will take this out and test again